src/client/cockpit-client-ssh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python3
```

Comment:
This would be the new Python script -- it needs to be added to the `__requires_exclude_from` in tools/cockpit.spec.

Reply:
I avoided unconditionally installing this.